src/pages/Home/home.jsx

```diff
@@ -0,0 +1,64 @@
+import "./home.css";
+import { Link } from "react-router-dom";
+
+const Home = () => {
+  return (
+    <>
+      <nav className="nav-container">
+        <div className="brand-name">
+          Pro<span style={{ color: "black" }}>NOTES</span>{" "}
+        </div>
+        <div className="nav-input">
+          <input placeholder="Search items" type="search" />
+        </div>
+        <div className="nav-cta">
+          <a className="primary-btn" href="">
+            Login
+          </a>
+        </div>
+      </nav>
+
+      <div className="child-wrapper">
+        <aside className="aside-wrapper">
+          <ul className="aside-list">
+            <li className="aside-item aside-item-active">
+              {" "}
+              <i class="fa-solid fa-house aside-icons"></i>{" "}
+              <a className="aside-link active" href="/">
+                Home
+              </a>
+            </li>
```

Comment:
Suggestion: You can move the sidebar into a separate component as it is going to be used on all the pages